### ISSUE TITLE
Add an About option in the Help menu

### ISSUE THREAD

**Issue by darellanodev** [darellanodev/yt-dlp-gui#8]
![image](https://github.com/user-attachments/assets/7904aa8c-1d83-4316-a60c-8e1b63713857)

Put the text:
This is a simple GUI for the yt-dlp application, built as a learning project to practice building desktop applications with Electron

**Comment by KiRZeN24** [comment]
Hi, I'm interested in this issue, can you assign it to me?

**Comment by darellanodev** [comment]
Yes, you can try resolving this issue, take the necessary time you need. I am assigning this issue to you.
